update VCS doc (#95)
SpinalHDL/SpinalDoc-RTD: source/SpinalHDL/Simulation/install/VCS.rst

  export LD_LIBRARY_PATH=$VERDI_HOME/share/PLI/Ius/LINUX64:$LD_LIBRARY_PATH 
  export LD_LIBRARY_PATH=$VERDI_HOME/share/PLI/MODELSIM/LINUX64:$LD_LIBRARY_PATH 

> For those who encounter the problem of ``Compilation of SharedMemIface.cpp failed``, make sure that you have installed C++ boost library correctly.

Reviewer: This is just nitpicking. So please chose witchever version to apply.
```suggestion
If you encounter the ``Compilation of SharedMemIface.cpp failed`` error, make sure that you have installed C++ boost library correctly.
```

> The header files and the library files path are added to ``CPLUS_INCLUDE_PATH``, ``LIBRARY_PATH`` and ``LD_LIBRARY_PATH`` respectively.

Reviewer: Maybe i understood the frase incorrectly. Is my suggestion correct? If so evaluate what version do you prefer :D
```suggestion
The header and library files path should be added to ``CPLUS_INCLUDE_PATH``, ``LIBRARY_PATH`` and ``LD_LIBRARY_PATH`` respectively.
```

Author: Great! Thanks for your suggestion.
